TimberAPI/EntityLinkerSystem/EntityLink.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace TimberbornAPI.EntityLinkerSystem
+{
+    /// <summary>
+    /// Represent a link between two entities
```

Comment:
Can we add a description of what you can use this for here?

Reply:
absolutely